lib/charms/tempo_k8s/v0/charm_instrumentation.py

```diff
@@ -200,10 +209,10 @@ def _get_server_cert(server_cert_getter, self, charm):
 
 
 def _setup_root_span_initializer(
-    charm: Type[CharmBase],
-    tracing_endpoint_getter: _GetterType,
-    server_cert_getter: Optional[_GetterType],
-    service_name: Optional[str] = None,
+        charm: Type[CharmBase],
```

Comment:
@PietroPasotti  the indentations seems off, curious how this passed `lint`.

Reply:
umh, not sure what happened there. Either way that library should be replaced by charm_tracing.py